server/src/main/java/org/mixit/cesar/cfp/model/ProposalVoteParam.java

```diff
@@ -0,0 +1,25 @@
+package org.mixit.cesar.cfp.model;
+
+public class ProposalVoteParam {
+
```

Comment:
For example https://github.com/mix-it/cesar/tree/master/server/src/main/java/org/mixit/cesar/site/web/api/dto
I put dto folders in the web packages